modules/terraform/options.go

```diff
@@ -66,29 +66,15 @@ type Options struct {
 }
 
 // Clone makes a deep copy of most fields on the Options object and returns it.
+//
+// NOTE: options.SshAgent and options.Logger can NOT be deep copied (e.g., the SshAgent struct contains channels and
```

Comment:
NIT: cannot is a single word :)

Reply:
Fixed: https://github.com/gruntwork-io/terratest/pull/693/commits/cf4757dde9c78bc741f7f49137ec2c63a42e1540